cmake/modules/RootConfiguration.cmake

```diff
@@ -620,7 +620,13 @@ get_filename_component(altcxx ${CMAKE_CXX_COMPILER} NAME)
 get_filename_component(altf77 "${CMAKE_Fortran_COMPILER}" NAME)
 get_filename_component(altld ${CMAKE_CXX_COMPILER} NAME)
 
-set(pythonvers ${PYTHON_VERSION_STRING})
+if(NOT PYTHON_VERSION_STRING_Development_Other)
+  set(pythonvers ${PYTHON_VERSION_STRING_Development_Main})
+else()
+  set(pythonvers "It is a multi-python build, please check --python2-version or --python3-version output")
```

Comment:
Please just report the main version here, otherwise people with scripts using `root-config --python-version` will break.

Reply:
@Axel-Naumann your opinion? (it is from our morning discussion)

Reply:
Actually it make sense, so lets do it like this (I want to land it quicker)

Reply:
@amadio how is "the main version" relevant to users of `--python-version`?

Reply:
Please see https://sft.its.cern.ch/jira/browse/ROOT-10293 and https://github.com/root-project/root/commit/43e0abba286df8fa61d0adc279bf7511ff05129a. The output of `root-config --python-version` and what CMake exports need to stay consistent.

Reply:
@amadio thanks a lot for help! I adjusted in 660eb28b38522f3438193cec741b5aaafaeb4af6